CONTRIBUTING.md

```diff
@@ -1,27 +1,146 @@
 # Contributing
 
-Your contributions are always welcome!
+NIST produces a lot of software, with open source code available
+on [GitHub][_usng]. If you believe one of our repositories is
+[*awesome*](AWESOME.md), please nominate it for inclusion in this
+[list][_awsm].
 
 ## Nominations
-[usnistgov](https://github.com/usnistgov) contains a lot of great
-code, and we welcome suggestions if you know of a repository that
-meets most of our [criteria](https://github.com/usnistgov/awesome-nist/blob/master/.github/pull_request_template.md).
-
-To nominate a repository for inclusion in the list, please file an 
-[issue](https://github.com/usnistgov/awesome-nist/issues) labeled
-"consider repository-name" (in lower case) with a link to the
-repository as the issue comment.
-
-## Guidelines
-* Search previous suggestions before making a new one, as yours may be a duplicate.
-* Add only one link per Pull Request.
-* Add the link: `* [project-name](https://github.com/usnistgov/REPOSITORY-NAME) - A short description ends with a period.`
-    * Keep descriptions concise.
-* Add a section if needed.
-    * Add the section description.
-    * Add the section title to Table of Contents.
-* Write `Closes #__` in the Pull Request description, referring to the associated Issue.
-* Fill out the Pull Request checklist as completely as possible. It should take 10 minutes or less to complete.
+
+To nominate a repository, file an [Issue][_issu] labeled "consider
+repository-name" (in lower case) with a link in the comment field.
+
+It would help peer review of the repository if you also paste the
+[checklist](AWESOME.md) into the comment field, and check off
+whatever applies. Don't worry if several boxes go unchecked:
+it gives the developers room to grow!
+
+## Guidelines for Contributors
+
+To add a repository to the curated list, please
+
+* Locate the [Issue][_issu] in which the repository was nominated,
+  or create a new Issue if one does not already exist.
```

Comment:
What function does an issue serve that isn't handled by just creating the pull request?